lib/cli.ex

```diff
@@ -4,11 +4,15 @@ defmodule Onigumo.CLI do
   }
 
   def main(argv) do
-    case OptionParser.parse(argv, strict: []) do
-      {[], [component], []} ->
+    case OptionParser.parse(
+           argv,
+           aliases: [C: :working_dir],
```

Comment:
Did we agree on including the alias directly and not create a separate follow-up PR?

Reply:
I suppose yes and we agree that the change is so small (one line in parser and two chars in usage message) that it is not necessary to make a whole new PR for that.